zo_haderech.py

```diff
@@ -0,0 +1,19 @@
+#submitted by Keren Or Hadad 208025205
+
+from os import listdir as dirs
+
+# part 5.1
+def zo_haderech(path: str):
+    item = []
+    for filename in dirs(path):
```

Comment:
Are there ways to simplify your code? (For example with list comprehension, or with string methods (try to read about "startswith")

Reply:
done